### PR TITLE
feat(adamant): add `txFetchInfo`

### PR DESCRIPTION
**Pull request by bludnic**
https://trello.com/c/tGvN9PhE/417-bug-after-adm-sending-the-app-sends-infinite-requests

### CODE REVIEW

File: assets/general/adamant/info.json [hunk 1]
```diff
@@ -19,24 +19,31 @@
   "defaultOrdinalLevel": 0,
   "consensus": "dPoS",
   "blockTimeFixed": 5000,
+  "txFetchInfo": {
+    "newPendingInterval": 4000,
+    "oldPendingInterval": 4000,
+    "registeredInterval": 4000,
+    "newPendingAttempts": 5,
```
Comment (adamant-al) [inline assets/general/adamant/info.json:26]: Why did you add this?

```
    "newPendingAttempts": 5,
    "oldPendingAttempts": 5
```

Comment (bludnic) [reply assets/general/adamant/info.json]: removed

**Review by adamant-al — APPROVED** [review]

**Review by Andrey1029 — APPROVED** [review]

**Comment by adamant-al** [comment]
@bludnic Merge it.
